Missing objective data for quest 25839 - The Ultrasafe Personnel Launcher (Dun Morogh, Cataclysm Classic)

<!-- READ THIS FIRST

Hello, thanks for taking the time to report a bug!

Before you proceed, please verify that you're running the latest version of Questie. The easiest way to do this is via the Curseforge client, but you can also download the latest version here: https://www.curseforge.com/wow/addons/questie

Questie is one of the most popular Classic WoW addons, with over 30 million downloads. However, like almost all WoW addons, it's built and maintained by a team of volunteers.

If you'd like to help, please consider making a donation. You can do so here: https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=aero1861%40gmail%2ecom&lc=CA&item_name=Questie%20Devs&currency_code=USD&bn=PP%2dDonationsBF%3abtn_donate_LG%2egif%3aNonHosted

You can also help as a tester, developer or translator. To do so please join the Questie Discord here https://discord.gg/fYcQfv7

-->


## Description
<!-- Explain in detail what problem you encountered. Is a quest not showing or showing again or an objective not showing or a spawn missing? -->

Hi folks! While questing through Dun Morogh, I've run into an issue at Steelgrill's Depot! I get the following error:

[ERROR] Missing objective data for quest 25839 Use Ultrasafe Personnel Launcher to deploy to Frostmane Retreat

## Wowhead link
<!-- If this issue is about a quest/npc/item/... then please include a link to it from Wowhead: https://wowhead.com/classic or https://wowhead.com/wotlk or https://wowhead.com/cata -->

https://www.wowhead.com/cata/quest=25839/the-ultrasafe-personnel-launcher

## Screenshots
<!-- If you can, add a screenshot to help explaining the problem (e.g. of your map where a quest is not shown but the NPC has an ! above him). Simply drag and drop the image in this input field, no need to upload it to any other image platform. -->

Questie Error:
![image](https://github.com/user-attachments/assets/04d0f3fd-d6eb-4390-bf97-593301d7acf9)

Quest Description:
![image](https://github.com/user-attachments/assets/e23feab3-f84c-4923-b9d8-3992e8653996)

## Questie & WoW version
<!--
Which version of Questie are you using? You can find it by:
- 1. Hovering over the Questie Minimap Icon
- 2. looking at your Questie.toc file (open it with any text editor).
It looks something like this: "v5.9.0" or "## Version: 5.9.0".

Please also state if your quest issue is about Classic Era/SoM or Wotlk.
-->

Questie 10.3.2
WoW 4.4.0 (Cataclysm Classic)

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3281,6 +3281,7 @@ function CataQuestFixes.Load()
         },
         [25839] = { -- The Ultrasafe Personnel Launcher
             [questKeys.triggerEnd] = {"Use the Ultrasafe Personnel Launcher to deploy to Frostmane Retreat.", {[zoneIDs.DUN_MOROGH] = {{56.85,46.65}}}},
+            [questKeys.objectives] = {{{41398,nil,Questie.ICON_TYPE_INTERACT}}},
         },
         [25843] = { -- Tortolla's Revenge
             [questKeys.preQuestSingle] = {25372},

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3279,6 +3279,9 @@ function CataQuestFixes.Load()
             [questKeys.preQuestSingle] = {},
             [questKeys.preQuestGroup] = {25520,25807},
         },
+        [25839] = { -- The Ultrasafe Personnel Launcher
+            [questKeys.triggerEnd] = {"Use the Ultrasafe Personnel Launcher to deploy to Frostmane Retreat.", {[zoneIDs.DUN_MOROGH] = {{56.85,46.65}}}},
+        },
         [25843] = { -- Tortolla's Revenge
             [questKeys.preQuestSingle] = {25372},
             [questKeys.nextQuestInChain] = 25904,

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3280,7 +3280,6 @@ function CataQuestFixes.Load()
             [questKeys.preQuestGroup] = {25520,25807},
         },
         [25839] = { -- The Ultrasafe Personnel Launcher
-            [questKeys.triggerEnd] = {"Use the Ultrasafe Personnel Launcher to deploy to Frostmane Retreat.", {[zoneIDs.DUN_MOROGH] = {{56.85,46.65}}}},
             [questKeys.objectives] = {{{41398,nil,Questie.ICON_TYPE_INTERACT}}},
         },
         [25843] = { -- Tortolla's Revenge

